fhservices.cfg.j2

```diff
@@ -121,6 +121,7 @@ define service {
 
 define service {
        service_description Storage::Pod::Disk Usage
+       # REVIEW: is this being applied to every pod/container? How does it work?
```

Comment:
At the nagios config level, only one service is being defined.  That service runs a check which dynamically finds all the pods to run the check command to summarize the results.  The overall status returned to nagios is based on the "worst" status of underlying checks (order UNKNOWN, CRIT, WARN, OK).